platform/ios/test/MGLMapViewConversionTests.m

```diff
@@ -0,0 +1,52 @@
+#import <Mapbox/Mapbox.h>
+#import <XCTest/XCTest.h>
+
+#import "../../ios/src/MGLMapView.h"
+
+@interface MGLMapViewConversionTests : XCTestCase
+
+@property (nonatomic) UIView *superView;
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation MGLMapViewConversionTests
+
+- (void)setUp {
+    [super setUp];
+    
+    [MGLAccountManager setAccessToken:@"pk.feedcafedeadbeefbadebede"];
+    self.superView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 64, 64)];
+    self.mapView = [[MGLMapView alloc] initWithFrame:CGRectMake(0, 0, 64, 64) styleURL:MGLStyle.streetsStyleURL];
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(33, 179)];
+    
+    [self.superView addSubview:self.mapView];
+}
+
+- (void)testCoordinateBoundsConversion {
+    MGLCoordinateBounds leftAntimeridianBounds = MGLCoordinateBoundsMake(CLLocationCoordinate2DMake(-75, 175), CLLocationCoordinate2DMake(75, 180));
+    CGRect leftAntimeridianBoundsRect = [self.mapView convertCoordinateBounds:leftAntimeridianBounds toRectToView:self.mapView];
+    
+    MGLCoordinateBounds rightAntimeridianBounds = MGLCoordinateBoundsMake(CLLocationCoordinate2DMake(-75, -180), CLLocationCoordinate2DMake(75, -175));
+    CGRect rightAntimeridianBoundsRect = [self.mapView convertCoordinateBounds:rightAntimeridianBounds toRectToView:self.mapView];
+    
+    MGLCoordinateBounds spanningBounds = MGLCoordinateBoundsMake(CLLocationCoordinate2DMake(24, 140), CLLocationCoordinate2DMake(44, 240));
+    CGRect spanningBoundsRect = [self.mapView convertCoordinateBounds:spanningBounds toRectToView:self.mapView];
+    
+    BOOL leftIntersection = CGRectIntersectsRect(spanningBoundsRect, leftAntimeridianBoundsRect);
+    BOOL rightIntersection =
+    CGRectIntersectsRect(spanningBoundsRect, rightAntimeridianBoundsRect);
+    
+    // If the resulting CGRect from -convertCoordinateBounds:toRectToView:
+    // intersects the set of bounds to the left and right of the
+    // antimeridian, then we know that the CGRect spans across the antimeridian
+    XCTAssertTrue(leftIntersection);
```

Comment:
Inline the call to `CGRectIntersectsRect()` into this line, so that any failure message will be more descriptive. Also, replace the comment with an assertion message that says the opposite in the event of a failure.